packages/server/modules/core/migrations/20240710154658_user-emails-backfill.ts

```diff
@@ -0,0 +1,26 @@
+import { UserRecord } from '@/modules/core/helpers/types'
+import { Knex } from 'knex'
+import crs from 'crypto-random-string'
+
+export async function up(knex: Knex): Promise<void> {
+  const users = await knex<UserRecord>('users').select(['id', 'email', 'verified'])
```

Comment:
You definitely need to do batching here, you're just retrieving all users at once into memory, while there could be thousands of them.

Pull them in batches of ~1000 or something